exoscale/resource_exoscale_compute.go

```diff
@@ -688,6 +707,16 @@ func resourceComputeUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange("reverse_dns") {
+		commands = append(commands, partialCommand{
+			partial: "reverse_dns",
+			request: &egoscale.UpdateReverseDNSForVirtualMachine{
+				DomainName: d.Get("reverse_dns").(string),
```

Comment:
Does `UpdateReverseDNSForVirtualMachine` accepts an empty value or should you use `DeleteReverseDNSFromVirtualMachine` instead? (I cannot recall)

Reply:
Good catch! Yes we should use `deleteReverseDnsFromVirtualMachine`, I'll make a patch. Thanks

Reply:
https://github.com/terraform-providers/terraform-provider-exoscale/pull/57/files#diff-c84e1274a05231e80d5b61ed7b5bd356